spec/rubocop/cop/style/string_literals_spec.rb

```diff
@@ -194,6 +194,30 @@
       expect(cop.config_to_allow_offenses).to eq('Enabled' => false)
     end
 
+    it 'registers offense for escaped single quote in single quotes' do
+      inspect_source(%q('\''))
+      expect(cop.messages)
+        .to eq(['Prefer double-quoted strings unless you need ' \
+                'single quotes to avoid extra backslashes for ' \
+                'escaping.'])
+    end
+
+    it 'does not accept multiple escaped single quotes in single quotes' do
+      inspect_source(%q('\'\'\''))
+      expect(cop.messages)
+        .to eq(['Prefer double-quoted strings unless you need ' \
+                'single quotes to avoid extra backslashes for ' \
+                'escaping.'])
+    end
+
+    it 'does not accept for multiple escaped single quotes in single quotes' do
```

Comment:
What's the point of this final example?

Reply:
@Drenmi asked for "a test case for multiple escaped single quotes (and no other escaped characters.)". I can add some more words to the string to make it look less forced if you prefer.

Reply:
I have changed the test string to a complete sentence.